lib/constants/color.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+class Colour {
+  Color lightPrimaryColor = const Color(0xffDF0054);
+  Color darkPrimaryColor = const Color(0xff480032);
+  Color secondaryColor = const Color(0xffFF8B6A);
+  Color accentColor = const Color(0xffFFD2BB);
```

Comment:
Choose one between Colour or Color

Reply:
then separate light and dark themes:
```
class LightThemeColor {...}
class DarkThemeColor {...}
class Color {...} // For Global
```

Reply:
> Choose one between Colour or Color

for class name is Colour because class name can't same with Color (flutter class object)